game/src/js/scenes/TitleScene.js

```diff
@@ -1,44 +1,60 @@
 import ServiceWorker from 'util/ServiceWorker';
 import Checkbox from 'objects/ui/Checkbox';
+import Button from 'objects/ui/Button'
+
 
 export default class TitleScene extends Phaser.Scene {
 
   constructor() {
     super({key: 'TitleScene'});
   }
-
+  
   preload() {
   }
-    
+
   create() {
-    this.add.image(400, 300, 'sky');
+    this.add.image(400, 430, 'sky');
```

Comment:
We should try to start making the game more resolution independent. Maybe we can calculate coordinates based on the size of the game? This is fine for now, but maybe the title and the buttons should be centered.